tests/cnf/core/network/dpdk/tests/rootless.go

```diff
@@ -492,15 +512,19 @@ var _ = Describe("rootless", Ordered, Label(tsparams.LabelSuite), ContinueOnFail
 				deploymentPod.Object.Annotations["k8s.v1.cni.cncf.io/network-status"])
 			Expect(err).ToNot(HaveOccurred(), "Fail to collect PCI addresses")
 
-			rxTrafficOnClientPod(deploymentPod, defineTestPmdCmd(tapOneInterfaceName, pciAddressList[0]))
+			err = cmd.RxTrafficOnClientPod(deploymentPod, defineTestPmdCmd(tapOneInterfaceName, pciAddressList[0]))
+			Expect(err).ToNot(HaveOccurred(), fmt.Sprintf(
```

Comment:
Remove fmt.Sprintf, the error will shown anyway

Reply:
removed